### PR TITLE
pAI Tweaks, Holochassis, & Radials

### PR DESCRIPTION
**Pull request by Skies-Of-Blue**
## About The Pull Request
Various ports, per request:
https://github.com/BeeStation/BeeStation-Hornet/pull/4917 Crow holochassis! (mycophobes stay winning)
https://github.com/BeeStation/BeeStation-Hornet/pull/4944 Puppy holochassis! 
https://github.com/BeeStation/BeeStation-Hornet/pull/5601 A smattering of holochassis (no Bee or Ghost)! + inhands for all holochassis. Rejoice!
https://github.com/tgstation/tgstation/pull/52879 Radials for holochassis selection!
https://github.com/BeeStation/BeeStation-Hornet/pull/8269 Radials for screen selection!
https://github.com/BeeStation/BeeStation-Hornet/pull/8977 and a final mothroach holochassis, for good luck.

Credit to Crushtoe, MistakeNot4892, and Pirill for the spritework!

## Why It's Good For The Game
These poor pAI players have been feeding on scraps for so long. Did you know that less than half of our available holochassis were capable of being held by players? No longer! Also swaps screen+chassis selection over from lists to radials, because having a preview of what you're selecting is nice, and it's 2024.

Here are the radials in action, as well as a teaser for a few of the new holochassis:
pAI Mains Eating Good Tonight

<|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|><|>

## Changelog

:cl:
imageadd: several new pAI holochassis
imageadd: held sprites for every existing pAI holochassis
tweak: all pAI holochassis can now be held
tweak: pAI screen and holochassis selections are now radials instead of lists
spellcheck: several instances of code are now feeling less michevious
:cl:

### CODE REVIEW

**Comment by thgvr** [comment]
pls no goon bee

Spider's kind of meh too, it's huge and got weird colors. Same for mushroom

**Comment by Skies-Of-Blue** [comment]
The bee here is explicitly not from goon, in fact. That was mentioned in the original PR. I can remove it if needed, however!

File: code/modules/mob/living/silicon/pai/pai_shell.dm [hunk 1]
```diff
@@ -72,11 +72,18 @@
...
-		return FALSE
-	var/choice = input(src, "What would you like to use for your holochassis composite?") as null|anything in sortList(possible_chassis)
+	var/list/skins = list()
+	for(var/holochassis_option in possible_chassis)
+		var/image/item_image = image(icon = src.icon, icon_state = holochassis_option)
+		skins += list("[holochassis_option]" = item_image)
+	sortList(skins)
+
+	var/atom/anchor = get_atom_on_turf(src)
+	var/choice = show_radial_menu(src, anchor, skins, custom_check = CALLBACK(src, .proc/check_menu, anchor), radius = 40, require_near = TRUE)
```
Comment (thgvr) [inline code/modules/mob/living/silicon/pai/pai_shell.dm]:
```suggestion
	var/choice = show_radial_menu(src, anchor, skins, custom_check = CALLBACK(src, PROC_REF(check_menu), anchor), radius = 40, require_near = TRUE)
```

**Comment by thgvr** [comment]
After going through the sprites I'd say to axe spider, walking mushroom, the bee(i know what the pr said but it looks exactly like goon) and the destiny one

**Comment by Sweettoothart** [comment]
The spider one is really good though?? PAI's need more options and spider is a really good one. Its green and black too?

**Comment by thgvr** [comment]
image
Here it's zoomed in for reference. It's the tg spider sprite with a thick dark blue outline and some colors swapped. It's also too large imo

**Comment by Skies-Of-Blue** [comment]
Full admission, the ghost wasn't axed purely out of laziness. I'll definitely be sure to remove that, the mushroom, and the Not!Goon Bee. No big losses there. 

The spider, however, is a darling of mine. Size-wise it's quite similar to the hawk or the fox, only marginally larger. When playing as a pAI, it's great to have a visual Notice Me! option, and the spider's color/size combo fill that niche well. It's far from our ugliest sprite, too; the fox chassis is both massive and crusty. I would prefer to keep the spider in its current state, if at all possible. 

I'll address any necessary changes tomorrow.

edit: comment was written before I noticed thgvr's response. oops.

**Comment by thgvr** [comment]
Spider needs to have a smaller design and more in line colors with the rest of them. The fox sucks, but it's old so whatever. I know hawk is quite big but it's not a recolored enemy sprite. Also, the snake's movement state's first icons for n/s are raised up too high, so it looks like its jumping.

PAIs could really use some sprite love honestly, but that's certainly not a job i'll push onto someone. I'm also 99% sure the dead states aren't used because when a pai "dies" it just turns back into a card. You could remove those if you'd like.

**Comment by thgvr** [comment]
Even if you just change the outline color to the same one as snake/frog it'd be better

**Comment by rye-rice** [comment]
oh my god, thank you so much for making the screens a radial!

is there a way to display the screen underlaying the pAI radial? its a bit hard to see the emoticons just from the screen overlays

**Comment by Skies-Of-Blue** [comment]
> oh my god, thank you so much for making the screens a radial!
> 
> is there a way to display the screen underlaying the pAI radial? its a bit hard to see the emoticons just from the screen overlays

That's probably beyond my very limited abilities, but I'll give it a shot!

**Comment by Skies-Of-Blue** [comment]
Unless any other issues come up, this should be ready for TM!

**Review by rye-rice — APPROVED** [review]
RADIALS ‼️

**Pull request merged**